tests/unit/test_task_proxy.py

```diff
@@ -101,3 +103,18 @@ def test_status_match(status_str: Optional[str], expected: bool):
     mock_itask = Mock(state=Mock(status='waiting'))
 
     assert TaskProxy.status_match(mock_itask, status_str) is expected
+
+
+@pytest.mark.parametrize('itask_flow_nums, flow_nums, expected', [
+    param({1, 2}, {2}, {2}, id="subset"),
+    param({2}, {1, 2}, {2}, id="superset"),
+    param({1, 2}, {3, 4}, set(), id="disjoint"),
+    param({1, 2}, set(), {1, 2}, id="all-matches-num"),
+    param(set(), {1, 2}, set(), id="num-doesn't-match-none"),
```

Comment:
The apostrophe will make running these tests individually from the shell a nightmare.